### PR TITLE
tests: re-include ObsPy

### PR DESCRIPTION
**Pull request by rstyd**
This PR addresses #64.

### CODE REVIEW

**Comment by rstyd** [comment]
This current version isn't quite working because it tries to write to a directory as stated in the test script. Do we still have a hard requirement for all the tests to run without internet access? I think it might be easier if we could relax that for this test.

**Comment by reidpr** [comment]
Pushed a few changes. In particular this now sidesteps the test suite problems by not running them. I'm not sure about all this. Please look it over and see what you think.

**Review by reidpr — COMMENTED** [review]
Pushed a few changes. In particular this now sidesteps the test suite problems by not running them. I'm not sure about all this. Please look it over and see what you think.

File: examples/obspy/test.bats [hunk 1]
```diff
@@ -6,13 +6,21 @@ load "${CHTEST_DIR}/common.bash"
...
 }
 
 @test "${ch_tag}/hello" {
-    ch-run -b "$BATS_TMPDIR":/mnt "$ch_img" -- /hello.py
+    ch-run -b "$inbind" -b "$outbind" "$ch_img" -- sh -c "/hello.py \
+                                                   /mnt/1/obspy.png"
 
     # Validate the plot is large enough for non-trivial content. (When I wrote
     # this test, it was 74KiB.) Leave it for manual examination if desired.
     [[ $(stat -c '%s' "$BATS_TMPDIR"/obspy.pdf) -gt 65536 ]]
```
Comment (reidpr) [inline examples/obspy/test.bats:22]: This won't pass any more since it's not creating the PDF. Note the full scope — maybe that's a mistake.

**Review by reidpr — APPROVED** [review]
A little more tidy. LMK whether you think it's ready for merge.

**Comment by rstyd** [comment]
Looks good to me!

**Comment by reidpr** [comment]
@jkmacc-LANL — FYI

**Pull request merged**